extensions/functions_arithmetic_decimal.yaml

```diff
@@ -102,6 +102,18 @@ scalar_functions:
           scale_after_borrow = max(init_scale - delta, min_scale)
           scale = init_prec > 38 ? scale_after_borrow : init_scale
           DECIMAL<prec, scale>
+  - 
+    name: "abs"
+    description: Calculate the absolute value of the argument.
+    impls:
+      - args:
+          - name: x
+            value: decimal<P,S>
+        options:
+          overflow:
```

Comment:
Can this operation overflow with decimal values?

Reply:
> Can this operation overflow with decimal values?

fixed, thanks.